packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -664,6 +667,17 @@ contract BridgePool is MultiCaller, Testable, BridgePoolInterface, ERC20, Lockab
         emit BridgePoolAdminTransferred(msg.sender, _newAdmin);
     }
 
+    /**
+     * @notice Enable the bridge admin to change the decay rate at which LP shares accumulate fees. The higher this
+     * value, the faster LP shares realize pending fees.
+     * @param _newLpFeeRatePerSecond The new rate to set.
+     */
+    function changeLpFeeRatePerSecond(uint64 _newLpFeeRatePerSecond) public override nonReentrant() {
+        require(msg.sender == address(bridgeAdmin));
```

Comment:
nit: should we make this a modifier?
nit: this should have a require message?
nit: should the natspec say only the admin can call this?

Reply:
> nit: should we make this a modifier?

yeah, slightly improves readability let's do that.

> nit: this should have a require message?

It definitely should have a require message, will add.

> nit: should the natspec say only the admin can call this?

Will add